README.md

```diff
@@ -25,7 +29,95 @@ If neither credentials file and env vars are found Swing will fail.
 To create your access keys see the [official AWS documentation][aws-credentials-docs].
 To know more about the environment variables see [this other documentation][aws-env-vars-docs].
 
-# Necessary AWS IAM permissions
+## Configuration Examples
+
+### Using AWS Client
+
+Install aws-cli using your favourite package manager or windows installer and Run
+
+```
+$ aws configure
+AWS Access Key ID [None]: my-aws-key
+AWS Secret Access Key [None]: my-aws-secret-key
+Default region name [None]: 
+Default output format [None]:
+```
+
+### Using Environment Variables
+
+***Linux***
+
+```
+esport AWS_ACCESS_KEY_ID=my-aws-key 
+export AWS_SECRET_ACCESS_KEY=my-aws-secret-key 
+```
+
+For a persistent configuration copy above lines in your
+
+```~/.bashrc``` or ```~/.zshrc``` depending on your shell 
+
+***Windows***
+
+TODO
```

Comment:
On Windows with using Powershell:
```
$Env:AWS_ACCESS_KEY_ID = "<my-aws-key>"
$Env:AWS_SECRET_ACCESS_KEY= "<my-aws-secret-key>"
```

You'll probably need enough permissions to set them.